utils/maintenance.sh

```diff
@@ -0,0 +1,170 @@
+    -s|--stub     : update cycstub
+
+The following environment variables must be set for this script to function:
+
+     CYCLUS_DIR : Environment variable CYCLUS_DIR must be set to the cyclus repository directory.
+    CYCSTUB_DIR : Environment variable CYCSTUB_DIR must be set to the cycstub repository directory.
+
+The -d|--nuc_data option requires the rs.cred files found in the Accounts Metadata file on Google Drive.
+
+EOF
+
+}
+
+die() {
+
+    echo -n >&2 "ERROR: "
+    echo >&2 "$@"
+    usage
+    exit 1
+
+}
+
+
+update_pyne() {
+
+    echo "Updating PyNE..."
+    HERE=`pwd`
+
+    wget https://github.com/pyne/pyne/archive/develop.zip
+    unzip -d pyne develop.zip
+    cd pyne/pyne-develop
+    ./amalgamate.py -s pyne.cc -i pyne.h
+    cp pyne.* $CYCLUS/src
+    cd ../..
+    rm -rf pyne develop.zip
+
+    cd $HERE
+
+}
+
+update_nuc_data() {
+
+    echo "Uploading new nuc_data.h5..."
+    HERE=`pwd`
+
+    # nuc_data upload
+    nuc_data_make -o cyclus_nuc_data.h5 \
+        -m atomic_mass,scattering_lengths,decay,simple_xs,materials,eaf,wimsd_fpy,nds_fpy
+    python upload_nuc_data.py
+    rm -r build_nuc_data cyclus_nuc_data.h5
+
+    cd $HERE
+
+}
+
+update_smbchk() {
+    
+    echo "Confirming ABI stability...."
+    HERE=`pwd`
+
+    cd $CYCLUS/release
+    ./smbchk.py --update -t HEAD --no-save --check | grep "ABI stability has been achieved!"
+    ./smbchk.py --update -t $VERSION
+    
+    cd $HERE
+}
+
+update_stub() {
+
+    echo "Updating cycstub...."
+    HERE=`pwd`
+
+    cp $CYCLUS/tests/input/stub_example.xml $CYCSTUB/input/example.xml
+    cp $CYCLUS/stubs/stub_* $CYCSTUB/src/
+
+    cd $HERE
+
+}
+
+
+update_all() {
+
+    echo "
+*-----------------------------------------------------------------------------*
+You're almost done!
+
+Cyclus, Cycamore, and Cycstub release candidates still need to be committed and
+pushed upstream
+
+Once the candidate passes CI it should be ready to be merged into develop and
+master.
+*-----------------------------------------------------------------------------*
+"
+
+
+}
+
+# check input
+CYCLUS=${CYCLUS_DIR?"Environment variable CYCLUS_DIR must be set to the cyclus repository directory."}
+CYCSTUB=${CYCSTUB_DIR?"Environment variable CYCSTUB_DIR must be set to the cycstub repository directory."}
+
+VERSION=-1
+
+while [[ $# > 0 ]]
+do
+    key="$1"
+
+    case $key in
+        -p|--pyne)
+        UPDATE_PYNE=1
+        ;;
+        -d|--nuc_data)
+        UPDATE_DATA=1
+        ;;
+        -a|--abi_chk)
+        UPDATE_SMBCHK=1
+        ;;
+        -s|--stub)
+        UPDATE_STUB=1
+        ;;
+        -r|--release)
+        UPDATE_PYNE=1
+        UPDATE_DATA=1
+        UPDATE_SMBCHK=1
+        UPDATE_STUB=1
+        ;;
+        -v|--version)
+        VERSION="$2"
+        shift
+        ;;
+        *)
+        die "unknown option: $key"
+        ;;
+    esac
+    shift
+done
+
+if [[ "$VERSION"       == -1 ]]; then
```

Comment:
I submitted a PR to fix this.  It gives me warm fuzzies to have that kind of alignment, but I guess it doesn't really jive with any style guides.   Maybe I'll have to propose a change to those :) jk